airflow/ui/src/queries/useDagParsing.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { useQueryClient } from "@tanstack/react-query";
+
+import {
+  useDagParsingServiceReparseDagFile,
+  UseDagServiceGetDagDetailsKeyFn,
+  UseDagSourceServiceGetDagSourceKeyFn,
+} from "openapi/queries";
+import { toaster } from "src/components/ui";
+
+const onError = () => {
+  toaster.create({
+    description:
+      "Dag parsing request failed. There could be pending parsing requests yet to be processed.",
+    title: "Dag Reparsing request",
```

Comment:
```suggestion
      "Dag parsing request failed. There could be pending parsing requests yet to be processed.",
    title: "Dag Failed to Reparse",
```

We should also check the actual error message to see if this is an appropriate description.

Reply:
The dag parsing endpoint posts the entry in the database which is picked up by the scheduler or standalone dag processor. I noticed error regarding unique constraint error when dag processor or scheduler is down and there is already a request but we can change it to generic error or just relay the API error too.